e2e-tests/tests/helpers.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from "@playwright/test";
+
+const authenticate = async (page) => {
+  await page.goto("/");
+
+  // Expect a title "to contain" a substring.
+  await expect(page).toHaveTitle(/Token/);
+
+  // Expects the URL to contain c/main/token
+  await expect(page).toHaveURL(/.*token/);
+
+  const token = process.env.HEADLAMP_TOKEN || '';
+  expect(token).not.toBe('');
+
+  // Fill in the token
+  await page.locator("#token").fill(token);
+
+  // Click on the "Authenticate" button
+  await page.click('button:has-text("Authenticate")');
+  // Wait for the page to load
+  await page.waitForLoadState('load');
+};
+
+const servicesPage = async (page) => {
+  // Click on the "Network" button
```

Comment:
For my own interest... I wonder with these was it difficult to use `await page.goto("/");` to other pages? Did that not work, and that is why you are using click? Or are there other advantages to doing it this way?

Are these links are always there? I guess the sidebar is there mostly.

Reply:
> I wonder with these was it difficult to use await page.goto("/"); to other pages?

When we navigate to "/" page after authenticating it takes time for page to load. Also, I wanted to test click functionality of the app so 2 birds in one stone I guess. Nothing much to it other than this. 

> Are these links are always there? I guess the sidebar is there mostly.
 
Yes sidebar is there and I wait for page to load for other things to load.

Reply:
Thanks for the explanation.